dataservice/extensions/flask_indexd.py

```diff
@@ -192,22 +218,29 @@ def _update_all_acls(self, record):
         target record's rev
         """
         # Only use fields allowed by the indexd PUT schema
-        fields = ['urls', 'acl', 'file_name', 'version',
+        fields = ['urls', 'acl', 'authz', 'file_name', 'version',
                   'metadata', 'urls_metadata', 'rev']
 
         url = '{}{}/versions'.format(self.url, record.latest_did)
-        versions = self.session.get(url).json()
+        try:
+            versions = self.session.get(url).json()
+        except RequestException as err:
+            abort(500, "Problem updating record in Indexd")
         for version, doc in versions.items():
-            if doc['acl'] != record.acl:
+            if doc['acl'] != record.acl or doc['authz'] != record.authz:
```

Comment:
Will `acl` always be present in the response or are they planning to remove it completely at some point? If the latter, should we change to `old.get('acl')`